delta_go/coords.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Code from: https://github.com/tensorflow/minigo
+
+"""Logic for dealing with coordinates.
+
+This introduces some helpers and terminology that are used throughout Minigo.
```

Comment:
Imo all references to Minigo should be cut. It's not something our users know/care about